cli/src/main/java/org/wildfly/glow/cli/commands/AbstractCommand.java

```diff
@@ -34,5 +39,85 @@ public abstract class AbstractCommand implements Callable<Integer> {
     @CommandLine.Option(
             names = {Constants.VERBOSE_OPTION_SHORT, Constants.VERBOSE_OPTION}
     )
-    Optional<Boolean> verbose;
+    boolean verbose;
+
+    @CommandLine.Spec
+    CommandLine.Model.CommandSpec spec;
+
+    private PrintWriter stdout;
+
+    private PrintWriter stderr;
+    private CommandLine.Help.Ansi ansi;
+
+    @Override
+    public final Integer call() throws Exception {
+        // print main command usage
+        spec.commandLine().usage(System.out);
```

Comment:
@jamezp , the failing tests are caused by this line. We are not expecting any output when compact is enabled. More generally do we really want to print the usage each time a command is called?

Reply:
Oops, I thought I ran the tests locally, but I must have done it before I made this change.